wled00/FX_fcn.cpp

```diff
@@ -1242,8 +1247,9 @@ void WS2812FX::show(void) {
   // See https://github.com/Makuna/NeoPixelBus/wiki/ESP32-NeoMethods#neoesp32rmt-methods
   busses.show();
 
-  // return bus brightness to original value
-  if (newBri < _brightness) busses.setBrightness(_brightness);
+  // returning bus brightness to its original value is done in the next frame, as to not interfere with asyncronous show()
+  // TODO if it is safe, prefer to restore brightness here
+  //if (newBri < _brightness) busses.setBrightness(_brightness, true);
```

Comment:
RMT uses dual buffers (as @Makuna explained to me [here](https://github.com/Aircoookie/WLED/issues/3264#issuecomment-1638412940)) so it *is* safe to immediately update brightness.
It may not be so with I2S (which we rarely use TBH).